### ISSUE TITLE
not working in Yosemite

### ISSUE THREAD

**Issue by mipmip** [naotaka/ClipMenu#8]
It crashes with:
   01-10-14 21:47:14,395 ClipMenu[670]: CoreData: error: fatal: This NSManagedObjectContext was created on the main thread and illegally passed to a background thread.

**Comment by naotaka** [comment]
Thanks for the report.

Can you tell me which version of Yosemite you're using? Isn't this the issue #7?

**Comment by mipmip** [comment]
Sorry I, missed isseu #7. I use developer preview 8. 

I also found a fix. Remove all preferences and other stuff related to clipmenu in ~/Library and then start clipmenu again.

**Comment by naotaka** [comment]
Thanks for the follow-up report, @mipmip!

It's bad that Yosemite still has a compatibility issue with ClipMenu. It seems that I need to release a working product of Swift version under development faster.

**Comment by adenisovgit** [comment]
Hi, may be you already have some working beta to use with Yosemite?
Using ClipMenu many years and now like without hands :)

**Comment by tru2dagame** [comment]
It works in the Yosemite in my Mac. Maybe try a reinstall to see if it fixed.
It crashed when the Yosemite was beta before.

**Comment by adenisovgit** [comment]
What version do you use?
I use 0.4.3 from official site. ClipMenu starts, but crash as soon as I try to select element

**Comment by tru2dagame** [comment]
Hi @adenisovgit
The same as mine. 0.4.3 also from the official site.
It crashed when I used the Yosemite beta. 
And I didn't do anything when I upgrade the GM version, it works well!

**Comment by adenisovgit** [comment]
Reinstall. No effect. Starts, then any activity in ClipMenu - instant crash (just disappear).

**Comment by naotaka** [comment]
Sorry to confuse you, but 0.4.4a12 is latest release. You can download it here:
http://www.clipmenu.com/en/dev/

Brand new ClipMenu is in very early stage now, lacks snippets and action feature. But works fine on Yosemite. I'm planning to release it for alpha test about next month.

**Comment by naotaka** [comment]
Before reinstall, see this instruction: https://gist.github.com/naotaka/894410ea4feb8aeb15a6

**Comment by adenisovgit** [comment]
Ah, that helps, thought all settings and snippets was lost. But it works!
Thank you.

**Comment by adenisovgit** [comment]
Ah, actually it didn't help.
After first use - again crash :(

**Comment by naotaka** [comment]
Hmm, it's strange. Could you upload the crash log to Gist or something? You can find it at `~/Library/Logs/DiagnosticReports`.

**Comment by adenisovgit** [comment]
https://drive.google.com/file/d/0Bzsvl5KUBlCgME5wZ25qbDRqYUU/view?usp=sharing

**Comment by naotaka** [comment]
Thanks for the report. Did this happen when a keyboard shortcut were invoked? It seems that it's hard to investigate this because I can't reproduce this bug here.

If your needs are mostly recording and reusing clipboard history, I can give you the pre-alpha build I'm using. But if snippets and actions are the key features for you, I beg you to wait for a while.

**Comment by adenisovgit** [comment]
No, it happens after I choose any item in dropdown menu with mouse.
It would be helpfull to use pre-alpha. snippets are good, but history is
more important.

2014-10-21 10:45 GMT+04:00 Naotaka Morimoto notifications@github.com:

**Comment by adenisovgit** [comment]
Also - I use http://plentycom.jp/en/steermouse/
May be it has influence

2014-10-21 11:40 GMT+04:00 Andrey Denisov adenisov@gmail.com:

**Comment by naotaka** [comment]
ClipMenu 1.0 pre-alpha:
https://dl.dropboxusercontent.com/u/1140644/clipmenu/ClipMenu_pre-alpha.zip

Warning: this build is in very early development stage. Please be careful to use it.
- This release doesn't have setting preferences. You cannot change number of items to show in the menu. Only you can change is the keyboard shortcut to popup menu. Default shortcut key is `Cmd-F2`.
- There is no limit to store clipboard history. If you want to remove old history items, you need to do so by yourself from `History Window`.
- Data model would be changed in the future release, but data migration may not be provided in this stage.

This build should work on Mac OS X 10.10, however if you encounter the app crash or any problem, please let me know.

**Comment by adenisovgit** [comment]
Looks like it works without crashs for now.
Thanks
You could use me as beta tester.
I use history and snippets.

2014-10-21 14:25 GMT+04:00 Naotaka Morimoto notifications@github.com:

**Comment by naotaka** [comment]
Sounds great! I'll further develop other features soon.

**Comment by skull-squadron** [comment]
Not to "works for me" disregard a possible issue but one box w/ clean retail Yosemite install and 0.4.3 appear to work:
- it tracks clipboard changes.
- clicking drop down status menu icon copies to the clipboard correctly.

I hope it works out.

**Comment by adenisovgit** [comment]
Hi, everybody.
Should we wait for complete working release of ClipMenu?
I'm missing other features, beside just history :)

2014-11-04 3:20 GMT+01:00 Barry Allard notifications@github.com:

**Comment by adenisovgit** [comment]
Looks like the project is dead :(
Does anybody know the analog of ClipMenu?

**Comment by tobiasboyd** [comment]
@adenisovgit there are a lot of clipboard managers in the App Store these days, a few of which seem to match ClipMenu pretty well in terms of features (history, snippets, etc.). I see http://www.apprywhere.com/copy-em-paste.html looks pretty capable, for example.
